<launch>
	<arg name="veh"/>
	<remap from="virtual_mirror_node/image_in" to="camera_node/image/compressed"/>
	<remap from="virtual_mirror_node/image_out" to="virtual_mirror_node/topic"/>
	<node ns="$(arg veh)" pkg="virtual_mirror_eharbitz" type="virtual_mirror_node.py" name="virtual_mirror_node" output="screen">
	</node>
</launch>